### DISCUSSION TITLE
Welcome to vue-i18n Discussions!

### DISCUSSION THREAD

**Discussion by kazupon** [kazupon/vue-i18n#1426] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by callmeDDDDD** [comment]
hello,
i got one question for use i18n in vue3，like i click english, its change to english，but i click again，there is no return。

**Reply by huntJs** [reply]
hello,
i use i18n in vue3, and i have a question that  Whether change tools are supported placeholder {} or %{} to other placeholder such as ${}, thanks
